R/pkg/inst/tests/testthat/test_mllib.R

```diff
@@ -141,3 +141,17 @@ test_that("kmeans", {
   cluster <- summary.model$cluster
   expect_equal(sort(collect(distinct(select(cluster, "prediction")))$prediction), c(0, 1))
 })
+
+test_that("survreg vs survival::survreg", {
+  data <- list(list(4,1,0,0), list(3,1,2,0), list(1,1,1,0),
+          list(1,0,1,0), list(2,1,1,1), list(2,1,0,1), list(3,0,0,1))
```

Comment:
If we use R's included survival dataset, we need load library survival which may not installed. So just generate the dataset follow R's survival documents.